code-reviews/README.md

```diff
@@ -0,0 +1,9 @@
+---
+description: Guidelines to make code reviews fruitful for developers and reviewers alike.
+---
+
+# Code Reviews
+
+Code reviews play a crucial role in the elementary development process. Having other people verify your code not only improves the overall code quality, they serve an even bigger purpose: Code reviews provide an opportunity for us to learn from each other. This guide tries to lower the barrier of reviewing code by providing a common ground of best practices and philosophies which proved to be useful over the years.
+
+Remember: The roles of a developer and a reviewer are not exclusive - you can be both. And we highly encourage you to do so, because this leads to more diverse reviews which help us to improve the overall user experience of elementary. So please make yourself familiar with the guidelines for [developers](prepare-code-for-review.md) and for [reviewers](reviewing-code.md).
```

Comment:
```suggestion
Remember: The roles of a developer and a reviewer are not mutually exclusive - you can be both. And we highly encourage you to perform both roles, because this leads to more diverse reviews which help us to improve the overall user experience of elementary. So please make yourself familiar with the guidelines for [developers](prepare-code-for-review.md) and for [reviewers](reviewing-code.md).
```